app/actions/client.py

```diff
@@ -69,8 +67,8 @@ class Unites(pydantic.BaseModel):
     FDI: str
 
 
-class HistoryResponse(pydantic.BaseModel):
-    History: List[HistoryItem]
+class ConditionsResponse(pydantic.BaseModel):
+    conditions: List[ConditionsItem]
     unites: Unites
```

Comment:
@vgarcia13 is this name something that comes from the Vital Weather API?
It might be worth using the name `units` in the model, so that the misspelled word doesn't propagate any further.

Reply:
😮 I didn't notice that typo. My brain ignored it (or thought it was ok), I will update it

Thanks Chris!